dev-helpers/index.html

```diff
@@ -42,6 +42,9 @@
       window["SwaggerUIStandalonePreset"] = window["swagger-ui-standalone-preset"]
       // Build a system
       const ui = SwaggerUIBundle({
+        advancedFilter: {
+          enabled: true
+        },
```

Comment:
@tim-lai is this fine probably commited this by accident?

Reply:
@mathis-m I would set `enabled: false` at minimum. I think having the option in dev-helpers is okay. Thanks.